### DISCUSSION TITLE
Is it possible to "Enable custom filters" from policy management?

### DISCUSSION THREAD

**Discussion by sofiedotcafe** [uBlockOrigin/uBlock-issues#3455] [General] [answered]
Hey!

Is it possible to enable "Enable my custom filters" from policy management (like, Firefox ESR policies)?
If not, would this then be considered a "security hole"?

Kind regards,
Sofie

**Comment by gorhill** [comment, marked as answer]
Yes, add their URL to `toOverwrite.filterLists`, same as with stock lists, see <https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin:-configuration#tooverwrite>.

**Comment by MSCHdd** [comment]
Sorry, i found no other Place for urgend Warning!
this Site is not blocked in uBlock, not blocked in others.
Is a massive Crypto-Scam with many References in Youtube (Fake-Vids) and Social Media ( "XAI19K" / "XAI80K" and similar)
`https://xaipresales.org/xai/`
`https://xai19k.com/`
please safe the users with a update.

is it possible to install a >Report Button< in uBlock?

**Reply by gorhill** [reply]
> is it possible to install a >Report Button< in uBlock?

There is already one, it's the _chat_ icon in the popup panel: https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#report-an-issue-with-current-website

**Reply by MSCHdd** [reply]
ok ... was too inconspicuous  ;- )
